packages/generators/src/handlers/default.ts

```diff
@@ -150,7 +150,7 @@ export async function questions(
       break;
   }
 
-  if (isCSS) {
+  if (isCSS || cssType === "SASS") {
```

Comment:
I think we should generate it for `LESS` and `Stylus` too

Reply:
should work for all css types now.